lib/RISCVParser.mly

```diff
@@ -17,10 +17,11 @@
 
 module A=RISCVBase
 
+(* We consider memory order and IO as the same*)
 let tr_rw = function
   | "r" -> A.R
-  | "w" -> A.W
-  | "rw" -> A.RW
+  | "w" | "ow" -> A.W
+  | "rw" | "iorw" -> A.RW
```

Comment:
As a result of this coding `iorw` will be printed as `rw` and litmus will emit `rw` in place of `iorw`... Is this intended?

Reply:
On table 26.3 of the [reference](https://riscv.org/wp-content/uploads/2019/06/riscv-spec.pdf) I can see that `fence iorw,iorw` means `Fence on all memory and I/O`. I could not find a reference for the meaning of `ow`. Since we only deal with Memory in herd I treat `iorw` as `rw`. Is this ok?